python/GangaLHCb/Lib/GaudiRunApp/GaudiRunAppUtils.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import time
+from Ganga.Core.exceptions import GangaException
+from Ganga.Runtime.GPIexport import exportToGPI
+from Ganga.Utility.logging import getLogger
+from Ganga.Core.exceptions import ApplicationConfigurationError
+from .PythonOptsCmakeParser import PythonOptsCmakeParser
+
+logger = getLogger()
+
+def readInputData(optsfiles, app):
```

Comment:
This seems to be an almost exact copy of the function you added in `PythonOptsCmakeParser.py`. Could they not have been combined?